app/javascript/components/DropdownSelector.js

```diff
@@ -171,40 +189,62 @@ const DropdownSelector = ({ sectors, companies, selectedOption, defaultFilter =
             <div className="dropdown-selector__options-wrapper">
               {isFilterBySector && (
                 <div className="dropdown-selector__options">
-                  {options.map(option => (
-                    <div onClick={() => handleOptionClick(option)} className="dropdown-selector__option">
+                  {(options.length > 0 && options.map((option, i) => (
```

Comment:
no need to check if it's `> 0` - `options.length` will evaluate to `false` if it's null or 0

Reply:
i know what you mean but readability speaking I only want something to happen when that condition is true. If options was an object with a map function and a length property, say, a string, the sole condition of checking the property would pass whereas the condition for checking its length won't.
TL;DR it's like `i++` vs `i += 1` readability vs habits